lib/pushmi_pullyu/aip/downloader.rb

```diff
@@ -1,215 +1,184 @@
+require 'fileutils'
+require 'ostruct'
+require 'rdf'
```

Comment:
We are using RDF in this class (not via rdf/n3) lets require this and bring in the gem as a dependency (incase rdf/n3 does something weird one day)